src/main/java/view/app/CardSearchBarView.java

```diff
@@ -0,0 +1,92 @@
+package view.app;
+
+import java.awt.GridBagLayout;
+import java.awt.LayoutManager;
+import java.awt.event.KeyAdapter;
+import java.awt.event.KeyEvent;
+import java.util.EventListener;
+import java.util.EventObject;
+import java.util.HashSet;
+
+import javax.swing.JButton;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTextField;
+import javax.swing.border.Border;
+import javax.swing.border.EmptyBorder;
+
+import usecase.app.cardsearch.CardSearchInputData;
+import util.GridBagConstraintBuilder;
+
+public class CardSearchBarView extends JPanel {
```

Comment:
Nice implementation. We should think of another way to implement card visualization after searching